src/bookshelf/src/bksw_wm_base.erl

```diff
@@ -20,9 +20,10 @@
 -module(bksw_wm_base).
 
 %% Complete webmachine callbacks
--export([init/1,
+-export([finish_request/2,
+         init/1,
          is_authorized/2,
-         finish_request/2,
```

Comment:
we would still need this finish_request function to be exported

Reply:
Done.